torchaudio/datasets/yesno.py

```diff
@@ -43,25 +34,26 @@ class YESNO(Dataset):
             Whether to download the dataset if it is not found at root path. (default: ``False``).
     """
 
-    _ext_audio = ".wav"
-
-    def __init__(self,
-                 root: Union[str, Path],
-                 url: str = URL,
-                 folder_in_archive: str = FOLDER_IN_ARCHIVE,
-                 download: bool = False) -> None:
+    def __init__(
+        self,
+        root: Union[str, Path],
+        url: str = _RELEASE_CONFIGS["release1"]["url"],
+        folder_in_archive: str = _RELEASE_CONFIGS["release1"]["folder_in_archive"],
+        download: bool = False
+                ) -> None:
```

Comment:
@mthrok I need your help again here. I am not sure how to resolve the style issue here. I am not exactly sure how to resolve this. [This](http://pep8online.com/checkresult) is what I was using.

Reply:
flake8 is no happy when indentation for continuous line is 4 spaces, because that is visually same as logical indentation. This often happens when `if` statement spans multiple lines or function indentation like this.

I think indenting 4 more spaces will resolve the issue.

```python
def __init__(
        self,
        ...
) -> None:
    ...
```